agnoster.zsh-theme

```diff
@@ -122,10 +122,20 @@ prompt_status() {
 
 # Display current virtual environment
 prompt_virtualenv() {
+  local env='';
+
   if [[ -n $VIRTUAL_ENV ]]; then
+    env=$VIRTUAL_ENV
+  fi
+
+  if [[ -n $CONDA_DEFAULT_ENV ]]; then
```

Comment:
also, its good practice to wrap your variables in quotes, otherwise spaces might give you wrong results:

```
if [[ -n "$CONDA_DEFAULT_ENV" ]]; then
  env="$CONDA_DEFAULT_ENV"
elif [[ -n "$VIRTUAL_ENV" ]]; then
  env="$VIRTUAL_ENV"
fi
```